addons/modules/fnc_getStringPos.sqf

```diff
@@ -0,0 +1,39 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_getStringPos
+
+Description:
+    A function used to get the position of a string with a passed typeName.
+
+Parameters:
+    - _type (String)
+        "OBJECT", "GROUP", "ARRAY", "MARKER", "TASK", or ""
+    - _string (String)
+
+Example:
+    (begin example)
+    [1, "player"] call CBA_fnc_getStringPos;
```

Comment:
`1` in the example call is not an object, group, array, marker or an empty string